addons/mail/static/src/models/chat_window_manager/chat_window_manager.js

```diff
@@ -109,9 +109,14 @@ function factory(dependencies) {
         openThread(thread, {
             isFolded = false,
             makeActive = false,
-            notifyServer = true,
+            notifyServer,
             replaceNewMessage = false
         } = {}) {
+            if (!notifyServer && this.env.messaging.device.isMobile) {
+                notifyServer = false;
+            } else {
+                notifyServer = true;
+            }
```

Comment:
condition should be like "if notify === undefined => notifyServer = !this.env.messaging.device.isMobile"
(if it's not undefined, keep current value)

also the same has to be done in the 3 methods in chat window

Reply:
This is a general rule that it is important to remember: whenever we are talking about default value/absence of value in JS `undefined` has to appear somewhere.

When we do default shortcut like `a = a || default` (which is basically a short version of your condition, without the mobile stuff), which we actually do a lot by simplicity, it's on the assumption that `a` in the example doesn't take a falsy value. This will always fail when `a` can be `0`, `false`, and such.

Reply:
I made the changes before reading your comment and I'm glad to see they match. Thanks for (unnecessary for once) reminder ! It's always good to have